NSrtm.Core.xTests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
```

Comment:
Please look at other projects to see how assembly attributes are handled in this solution.